build/ballot/1/index.json

```diff
@@ -682,11 +682,11 @@
             "supporting_by_type": {
               "Print Ads": 2048.0,
               "CANVASSING": 5000.0,
-              "Phone Banks": 597.7,
+              "Phone Banks": 598.0,
               "ESTIMATE OF LIT": 7915.0,
-              "ESTIMATE OF STAFF TIME": 31088.83,
-              "Campaign Paraphernalia/Misc.": 2702.11,
-              "Campaign Literature and Mailings": 44502.69,
+              "ESTIMATE OF STAFF TIME": 31089.0,
+              "Campaign Paraphernalia/Misc.": 2702.0,
+              "Campaign Literature and Mailings": 44503.0,
```

Comment:
It looks like we lost resolution to the penny. I wonder if roo isn't handling floats properly?